physionet-django/search/templates/search/content_list.html

```diff
@@ -28,7 +28,7 @@
     <p class="pub-details">Published: {{ published_project.publish_datetime|date }}.
       Version: {{ published_project.version }}</p>
     {% if is_lightwave_supported %}
-      {% call_method published_project 'has_access' request.user as user_has_access %}
+      {% call_method published_project 'can_view_files' request.user as user_has_access %}
```

Comment:
Should `user_has_access` also be updated in this template? (e.g. to `user_can_view_files`)?